packages/vue-vanilla/package.json

```diff
@@ -90,7 +89,7 @@
     "tslib": "^2.5.0",
     "typedoc": "~0.25.3",
     "typescript": "~4.9.5",
-    "vue": "^3.2.26",
+    "vue": "^3.4.21",
```

Comment:
As we don't adjust the `peerDependencies`, did you test whether the built package is still compatible with a consumer using `vue` `3.2.26`?

Reply:
Good point. I now tested it with the vue seed by packaging the locally built jsonforms packages and installing them there. With this, it worked with vue 3.2.26 :)